docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md

```diff
@@ -145,19 +158,23 @@ Accept wildcard characters: False
 ```
 
 ### -MigrationType
-Specifies the type of migration to perform for the virtual machine.
-The options are as follows: 
 
- -- Live: Transparently migrates the virtual machine without a dropped network connection or perceived downtime. 
+Specifies the type of migration to perform for the virtual machine. The options are as follows:
+
+ -- Live: Transparently migrates the virtual machine without a dropped network connection or
+ perceived downtime.
```

Comment:
This suggestion (and the following) turn the non-standard syntax into a list - I would've done them in a single suggestion but the unedited lines in between made GitHub unhappy.

```suggestion
- `Live`: Transparently migrates the virtual machine without a dropped network connection or
  perceived downtime.
```